packages/core/src/lib/visit/visit.ts

```diff
@@ -79,6 +79,35 @@ export const visit: Visiter = ({ deps, node, metadata }): ts.Node => {
       ],
       deps: deps,
     });
+
+    // Append the JSDoc comments from the input node onto the output node
+    if (Object.keys(node).includes("jsDoc")) {
+      const getSrcFile: (node: ts.Node) => ts.SourceFile = (node) => {
```

Comment:
Can we please put this in the utils? This is just a pure function right? Ie. it's not a closure?

Reply:
Sure